packages/manager/modules/bm-server-components/src/consumption-tile/controller.js

```diff
@@ -0,0 +1,15 @@
+export default class QuotaTileController {
+  /* @ngInject */
+
+  canOrderTraffic() {
+    return !this.server.isExpired && this.server.canOrderQuota;
+  }
+
+  canOrderMoreTraffic() {
+    return (
+      !this.server.isExpired &&
+      this.server.canOrderQuota &&
+      this.trafficOrderables?.data.length
```

Comment:
```suggestion
      this.canOrderTraffic() &&
      this.trafficOrderables?.data.length
```

Reply:
I remove canOrderTraffic() because isn't use and the test is already includes in canOrderMoreTraffic().